recipes/perl-encode/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "perl-encode" %}
+{% set version = "3.02" %}
+{% set sha256 = "5865be4951870f62f43722818d076d7724306c75c8f268346b282351cbc820a8" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: Encode-{{ version }}.tar.gz
+  url: https://cpan.metacpan.org/authors/id/D/DA/DANKOGAI/Encode-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true    # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+  host:
+    - perl
+    - perl-storable
+    - perl-extutils-makemaker
+    - perl-exporter
+    - perl-parent
+    - make
+    #- perl-test-more
+
+  run:
+    - perl
+    - perl-storable
+    - perl-exporter
+    - perl-parent
+
+test:
+  imports:
+    - Encode
+  commands:
+    - perl -we 'use Encode::Alias'  # version does not match that of Encode
+
+about:
+  home: http://metacpan.org/pod/Encode
+  license:  GPL-1.0-or-later
+  license_file: gpl-1.0.txt
```

```suggestion
  license_file:
    - {{ environ["PREFIX"] }}/man/man1/perlartistic.1
    - {{ environ["PREFIX"] }}/man/man1/perlgpl.1
```

This is the pattern I've been using in other Perl recipes I've migrated from Bioconda; similar to R we can use license files that Perl already installs.